src/scenes/Root/Header/ProfileMenu/ProfileMenu.tsx

```diff
@@ -49,6 +53,8 @@ export const ProfileMenu = (props: Partial<MenuProps>) => {
       </Typography>
       <Divider {...skipAutoFocus} />
       <MenuItemLink to="/logout">Sign Out</MenuItemLink>
+      <MenuItem onClick={changePassword}>Change Password</MenuItem>
+      <ChangePassword {...changePasswordState} />
     </Menu>
```

Comment:
```suggestion

    </Menu>
    <ChangePassword {...changePasswordState} />
  </>
```

Including `<ChangePassword>` inside the `<Menu>` messes up the keyboard-tabbing in the form.

Reply:
I moved this outside the menu and made the other change to get rid of the margin and Grid elements for each field.